utils.py

```diff
@@ -17,9 +17,11 @@
 #
 # Version 3: run_description column is added to the runs table
 #
-# Version 4: is actually version 3 again, but has a separate upgrader
+# Version 4a: is actually version 3 again, but has a separate upgrader
 #   to fix bugs in how the run_description was written
 #
+# Version 4: snapshot column is made always present in the runs table
```

Comment:
This happens in version 5?

Reply:
i was frankly confused, but i think the format is "<version before the change>: <description of the change>". @WilliamHPNielsen am i right?

Reply:
This should perhaps read. `The upgrade from 4->5 ensures that the snapshot column is made always present in the runs table`

Reply:
Let me rewrite this whole paragraph as you suggested.

Reply:
also see slack discussion

Reply:
done